assets/scaffold/recipes/sous_admin/recipe.yml

```diff
@@ -0,0 +1,102 @@
+name: 'Sous Admin'
+description: 'A recipe that contains modules, themes, and configuration for the Sous administration experience.'
```

Comment:
@callinmullaney I updated the description because the original which said "Sous's" administration experience was causing an error on the single quote (apostrophe).